Jenkinsfile

```diff
@@ -136,25 +126,21 @@ pipeline {
       }
 
       environment {
-        GITHUB_PACKAGES_TOKEN = credentials('github-api-token')
-        NPM_PUBLISH_TOKEN = credentials('npm-publish-token')
+        GITHUB_TOKEN = credentials('github-api-token')
+        NPM_TOKEN = credentials('npm-publish-token')
         NPM_CONFIG_CACHE = '.npm'
         NPM_CONFIG_USERCONFIG = '.npm/rc'
         SPAWN_WRAP_SHIM_ROOT = '.npm'
+        GIT_AUTHOR_NAME = 'LogDNA Bot'
+        GIT_AUTHOR_EMAIL = 'bot@logdna.com'
+        GIT_COMMITTER_NAME = 'LogDNA Bot'
+        GIT_COMMITTER_EMAIL = 'bot@logdna.com'
```

Comment:
We should probably just load these into jenkins so we don't have to hard code them all over.

Reply:
thoughts on adding these globally, vs creating a `semanticRelease` groovy function which applies them a la `versioner`?

Reply:
ya. that works too. less copy + paste

Reply:
that was on the list of things to do, just haven't gotten there. it would be nice to have a `dry` option as well so we don't have to copy that part around

Reply:
Have y'all made a ticket for this? Releng can solve this globally without code.

Reply:
just thinking out loud, we also have the multi-release tool for mono-repos. would be nice to have that too. maybe a `release` name space or something

```
release.legacy(...opts) // points to versioner
release.standard(...opts) // run semantic-release
release.multi(...opts) // runs multi-release
```

Reply:
@gregswift how hard is it to at least get the env vars set globally?  Should we block this PR while we wait for that? I think we should continue the discussion re how to bundle things nicely within Jenkins as @esatterwhite mentions.

Reply:
created a card for getting these vars added to jenkins, but we'll leave it defined here too for now.

Reply:
Adding them to jenkins looks easy enough. I don't think we realized it wasn't setup.

Reply:
> We should probably just load these into jenkins so we don't have to hard code them all over.

looking at the jenkins repo, we are already putting in a global git username email:
https://github.com/answerbook/jenkins/blob/master/init.groovy.d/10-global-settings.groovy#L47

checking inside Jenkins Config UI, it seems those values differ from above. would making a PR to change those values resolve this?

<img width="702" alt="Screen Shot 2021-02-17 at 2 59 54 PM" src="https://user-images.githubusercontent.com/1727467/108261022-0966cb80-7131-11eb-9538-a0a1ae25015d.png">